typescript/examples/vercel-ai/lens/chain.ts

```diff
@@ -0,0 +1,25 @@
+import { defineChain } from "viem";
+
+export const lens_testnet = /*#__PURE__*/ defineChain({
```

Comment:
You can also use the lens package to import the network without having to define it :)

https://dev-preview.lens.xyz/docs/network/getting-started/viem